client/Cargo.toml

```diff
@@ -18,7 +18,7 @@ futures01 = { package = "futures", version = "0.1.29" }
 jsonrpsee = "0.1.0"
 log = "0.4.13"
 sc-network = { version = "0.8.1", default-features = false }
-sc-service = { version = "0.8.1", default-features = false }
+sc-service = { version = "0.8.1", default-features = false, features = ["wasmtime"] }
```

Comment:
`wasmtime` only works on `x86` last I checked and it could make sense to make it an optional feature instead or [just enable it on x86 as substrate does](https://github.com/paritytech/substrate/blob/master/bin/node/cli/Cargo.toml#L108-#L112)

Reply:
Good point, updated!